src/jit/compiler.h

```diff
@@ -3102,6 +3102,10 @@ class Compiler
 
     //---------------- Spilling the importer stack ----------------------------
 
+    // The maximum tree size in the importer before it spills all trees from the stack into local variables.
```

Comment:
"maximum tree size..." is a bit misleading, as this is actually "maximum number of bytes of IL processed..."